packages/admin/src/form/FinalFormRangeInput.tsx

```diff
@@ -54,8 +57,8 @@ const FinalFormRangeInputComponent: React.FunctionComponent<WithStyles<typeof st
     endAdornment,
     input: { name, onChange, value: fieldValue },
 }) => {
-    const [internalMinInput, setInternalMinInput] = React.useState(fieldValue.min || 0);
-    const [internalMaxInput, setInternalMaxInput] = React.useState(fieldValue.max || 0);
+    const [internalMinInput, setInternalMinInput] = React.useState(fieldValue.min || min);
+    const [internalMaxInput, setInternalMaxInput] = React.useState(fieldValue.max || max);
```

Comment:
https://github.com/vivid-planet/comet-admin/pull/406 slipped in here. If you really need it you have to properly depend on this PR

Reply:
i know i will remove i just used it to test my count Method :) thats why my commit is a WIP Commit so everybody can test it :)

Reply:
yes i know :) i copied it to test my count Method. hopefully my bug-fix in the other PR can soon be merged so i can rebase this PR with the next Branch :)